cli/volume.go

```diff
@@ -322,27 +324,39 @@ func (v *VolumeListCommand) runVolumeList(args []string) error {
 		return err
 	}
 
-	display := v.cli.NewTableDisplay()
-	displayHead := []string{"DRIVER", "VOLUME NAME"}
-	if v.size {
-		displayHead = append(displayHead, "SIZE")
+	if (v.size || v.mountPoint) && v.quiet {
+		return fmt.Errorf("Conflicting options: --size (or --mountpoint) and -q")
 	}
-	if v.mountPoint {
-		displayHead = append(displayHead, "MOUNT POINT")
+
+	display := v.cli.NewTableDisplay()
+	displayHead := []string{"VOLUME NAME"}
+
+	if !v.quiet {
+		displayHead = append([]string{"DRIVER"}, displayHead...)
```

Comment:
Just asking, can we wrap the logic of the `displayHead` ?

Reply:
wrap the logic into a function?